DigitalExcellence/dex-backend
# appsettings load in fail fix

## Description

Renamed appsettings to fix an issue where API loaded the wrong appsettings in docker environment

All appsettings now have unique names.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Run project using docker-compose up.
2. Check if all projects are running

**Please check if notification server starts. I had problems today, but earlier aswell, so it might be my machine.**

## Link to issue

Closes: #349
